Gruntfile.js

```diff
@@ -181,14 +181,16 @@ module.exports = function(grunt) {
       setupAdmin1: {
         cmd: 'curl -X PUT http://localhost:5984/_config/admins/admin -d \'"pass"\'' +
              ' && curl -X POST http://admin:pass@localhost:5984/_users ' +
-             ' -H "Content-Type: application/json" ' +
-             ' -d \'{"_id": "org.couchdb.user:admin", "name": "admin", "password":"pass", "type":"user", "roles":[]}\''
+                 ' -H "Content-Type: application/json" ' +
+                 ' -d \'{"_id": "org.couchdb.user:admin", "name": "admin", "password":"pass", "type":"user", "roles":[]}\' ' +
+             ' && curl -X PUT --data \'"true"\' http://admin:pass@localhost:5984/_config/couch_httpd_auth/require_valid_user'
```

Comment:
Yeah, shame that:
1. we don't have our real couch config in e2e tests
2. we don't run nginx in e2e tests

We should run these tests against a medix container.